.gitignore

```diff
@@ -33,6 +33,7 @@ ddtrace/internal/datadog/profiling/libdatadog/
 .Python
 env/
 build/
+cmake_build/
```

Comment:
This was recently removed by @sanchda in https://github.com/DataDog/dd-trace-py/commit/4489271007a802605b490f69e980b4344c9dbbc5#diff-bc37d034bad564583790a46f19d807abfe519c5671395fd494d8cce506c42947L36

Do we need it?

Reply:
yeah, not anymore, I'm gonna drop it